sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
We do have Sufia 7 releases out and people are using them. If someone's gotten their app upgraded to use `7.0.0.beta4` and they are using devise 3.5, I don't want to complicate their path to 7.0.0 proper. I don't disagree with where you're coming from, though.